packages/atlas/src/views/studio/MyPaymentsView/PaymentsTransactionTab/PaymentTransactionsTab.styles.ts

```diff
@@ -0,0 +1,13 @@
+import styled from '@emotion/styled'
+
+import { media, sizes } from '@/styles'
+
+export const TilesWrapper = styled.div`
+  display: grid;
+  gap: ${sizes(4)};
+  margin-bottom: ${sizes(4)};
```

Comment:
I just noticed that the margin and the gap are different on `md`. It should be 24px not `16 px`.  I did the same mistake in `Overview tab`. Could you fix it there as well? 
https://www.figma.com/file/ZFRXsyGvnBXnkDUUGc5vyR/My-payments-page?node-id=1505%3A17428&t=gGHo1JzBlZspPmLd-4